scripts/upgrade

```diff
@@ -79,13 +84,8 @@ ynh_add_nginx_config
 #=================================================
 ynh_script_progression --message="Upgrading dependencies..." --weight=12
 
-# Migration: remove old repository if defined
-if [ -f "/etc/apt/sources.list.d/grafana_stable.list" ] ; then
-	ynh_secure_remove --file="/etc/apt/sources.list.d/grafana_stable.list"
-fi
-
 ynh_install_app_dependencies $pkg_dependencies
-ynh_install_extra_app_dependencies --repo="deb https://packages.grafana.com/oss/deb stable main" --package="grafana (>=$GRAFANA_VERSION)" --key="https://packages.grafana.com/gpg.key"
+ynh_install_extra_app_dependencies --repo="deb https://packages.grafana.com/oss/deb stable main" --package="grafana" --key="https://packages.grafana.com/gpg.key"
```

Comment:
Hmm not sure that doesn't do harm... In my memories, it was to make sure the already installed package is upgraded, otherwise it just says it's already installed. Did you test it?

Reply:
I will, but with `(>=$GRAFANA_VERSION)` upgrade fails https://ci-apps-dev.yunohost.org/ci/job/4637

BTW, we are speaking of the upgrade of package when using `ynh_install_extra_app_dependencies`
https://github.com/YunoHost-Apps/zabbix_ynh/pull/32#discussion_r796738884

I will check if the package is upgraded...

Reply:
(Yeah I confirm that it should be okay, the helper explicitly install the latest version now (well sort of))